LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
License still BSD-3, but the copyright was changed from Anaconda Inc. to Sparse developers in order to acknowledge my own and others' contributions.

Reply:
Also changed to not be RST so GitHub can detect it.

Reply:
We probably need permission from @pzwang here.

Peter, this is a small project I did in my off time during employment at Anaconda Inc.  I hope that it becomes relevant at some point to the company's work, but these days it is mostly managed by others in the community, like @hameerabbasi .  It would be nice for the license to reflect that change.

Reply:
Thanks for the background.  This change looks good to me - and thanks to @hameerabbasi and others for working on the project!!